src/Utilities/FlowAnalysis/FlowAnalysis/Framework/DataFlow/DataFlowOperationVisitor.cs

```diff
@@ -3361,6 +3362,16 @@ public override TAbstractAnalysisValue VisitAwait(IAwaitOperation operation, obj
                 value;
         }
 
+        public override TAbstractAnalysisValue VisitReDimClause(IReDimClauseOperation operation, object? argument)
+        {
+            var value = base.VisitReDimClause(operation, argument);
+            if (PredicateAnalysis)
```

Comment:
ReDim clause can never be a predicate, this is not correct.

Reply:
Ok so what's the point of overriding the method in this class if the only thing is to return the base?